### PR TITLE
[Bulk] Validation: validation fixes JM 5/30

### PR DESCRIPTION
**Pull request by jmart1428**
This PR fixes 30 missing descriptions.

### CODE REVIEW

File: docs/outlook/mapi/cbnewflatmtsidlist.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "CbNewFLATMTSIDLIST"
- 
- 
+description: "CbNewFLATMTSIDLIST computes the number of bytes that should be allocated for a new FLATMTSIDLIST structure that contains several MTSID structures of a specified size."
```
Comment (dereklh7) [inline docs/outlook/mapi/cbnewflatmtsidlist.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: docs/outlook/mapi/cbnewspropattrarray.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "CbNewSPropAttrArray"
- 
- 
+description: "CbNewSPropAttrArray computes the number of bytes that should be allocated for a new SPropAttrArray structure that contains a specified number of property attributes."
```
Comment (dereklh7) [inline docs/outlook/mapi/cbnewspropattrarray.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: docs/outlook/mapi/cbnewspropproblemarray.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "CbNewSPropProblemArray"
- 
- 
+description: "CbNewSPropProblemArray computes the number of bytes to be allocated for a new SPropProblemArray structure that contains a specified number of SPropProblem structures."
```
Comment (dereklh7) [inline docs/outlook/mapi/cbnewspropproblemarray.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: docs/outlook/mapi/cbnewssortorderset.md [hunk 4]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "CbNewSSortOrderSet"
- 
- 
+description: "CbNewSSortOrderSet computes the number of bytes to be allocated for a new SizedSSortOrderSet structure that contains a specified number of sort orders represented by SSortOrder structures."
```
Comment (dereklh7) [inline docs/outlook/mapi/cbnewssortorderset.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: docs/outlook/mapi/cbspropattrarray.md [hunk 5]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "CbSPropAttrArray"
- 
- 
+description: "CbSPropAttrArray computes the number of bytes in an existing SPropAttrArray structure."
```
Comment (dereklh7) [inline docs/outlook/mapi/cbspropattrarray.md:3]: Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: docs/outlook/mapi/cbspropproblemarray.md [hunk 6]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "CbSPropProblemArray"
- 
- 
+description: "CbSPropProblemArray computes the number of bytes in an existing SPropProblemArray structure."
```
Comment (dereklh7) [inline docs/outlook/mapi/cbspropproblemarray.md:3]: Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: docs/outlook/mapi/commonly-used-header-files.md [hunk 7]
```diff
@@ -1,13 +1,13 @@
 ---
 title: "Commonly Used Header Files"
+description: "Describes the MAPI header files that developers frequently use."
```
Comment (dereklh7) [inline docs/outlook/mapi/commonly-used-header-files.md:3]: Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: docs/outlook/mapi/commonly-used-objects.md [hunk 8]
```diff
@@ -1,13 +1,13 @@
 ---
 title: "Commonly Used Objects"
+description: "Describes commonly used MAPI objects and interfaces."
```
Comment (dereklh7) [inline docs/outlook/mapi/commonly-used-objects.md:3]: Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: docs/outlook/mapi/commonly-used-properties.md [hunk 9]
```diff
@@ -1,13 +1,13 @@
 ---
 title: "Commonly Used Properties"
+description: "Lists of required and optional MAPI properties that are documented in several places."
```
Comment (dereklh7) [inline docs/outlook/mapi/commonly-used-properties.md:3]: Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: docs/outlook/mapi/contents-tables.md [hunk 10]
```diff
@@ -1,13 +1,13 @@
 ---
 title: "Contents Tables"
+description: "Describes contents tables, which contain information about objects in a MAPI container."
```
Comment (dereklh7) [inline docs/outlook/mapi/contents-tables.md:3]: Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: docs/outlook/mapi/content-retired-notice-mapi.md [hunk 11]
```diff
@@ -1,11 +1,11 @@
 ---
 title: "Content Retired Notice (MAPI)"
+description: "Content retired notice. Provides resources to Microsoft Outlook 2013 Messaging API content."
```
Comment (dereklh7) [inline docs/outlook/mapi/content-retired-notice-mapi.md:3]: Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: docs/outlook/mapi/copying-or-moving-a-message-or-a-folder.md [hunk 12]
```diff
@@ -1,13 +1,13 @@
 ---
 title: "Copying or moving a message or a folder"
+description: "Describes the four methods used to copy or move a message or a folder in Microsoft Outlook."
```
Comment (dereklh7) [inline docs/outlook/mapi/copying-or-moving-a-message-or-a-folder.md:3]: Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

**Comment by jmart1428** [comment]
**Global effort to fix build validation errors**
@lindalu-MSFT 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Pull request merged**